### ISSUE TITLE
[Github] Add support for !CAUTION/!IMPORTANT/!NOTE/!TIP/!WARNING using blockquote

### ISSUE THREAD

**Issue by joshuataylor** [facelessuser/MarkdownPreview#174]
GitHub has added support for NOTE/TIP/etc which display as a nice little blockquote.

See [An option to highlight a "Note" and "Warning" using blockquote (Beta) #16925](https://github.com/orgs/community/discussions/16925)

This is BETA, but I believe they've settled on the format. Might be worth holding off until it's stable though.

This would be awesome to support this as I really like it for README.md and other documentation.

edit: I believe this is an upstream issue, I'll take a look.

**Comment by joshuataylor** [comment]
I did a bit of digging, and I had no idea how all this worked, here is a summary from my limited understanding:

The Sublime package [MarkdownPreview](https://github.com/facelessuser/MarkdownPreview) uses a Python library called [Python-Markdown](http://github.com/Python-Markdown/markdown) to convert Markdown offline.

To add features that might not be supported, you can create an [extension](https://python-markdown.github.io/extensions/).

## Summary of Docs Alerts

1. There was a request to add [Add support for GitHub Flavored Markdown new admonitions](https://github.com/Python-Markdown/markdown/issues/1423) to this library, tl;dr: this should exist in [markdown-callouts](https://github.com/oprypin/markdown-callouts).
2. There is a request to [Add "Microsoft Docs Alerts" style Callouts](https://github.com/oprypin/markdown-callouts/issues/13) in the markdown-callouts extension - also see the [great comment](https://github.com/oprypin/markdown-callouts/issues/13#issuecomment-1858013783) summarising this.

So basically, either this will be added to markdown-callouts, or perhaps another extension for python-markdown.

**Comment by facelessuser** [comment]
It is not the Python Markdown parser that needs updating. You need to use the GitHub parse in the right mode, but it is likely the GitHub styling CSS needs to be updated.

**Comment by facelessuser** [comment]
I'm updating the GitHub styling, so it should style things correctly if you are using the GitHub parser in `gfm` mode:

<img width="689" alt="Screenshot 2024-01-14 at 1 21 34 PM" src="https://github.com/facelessuser/MarkdownPreview/assets/1055125/10b453ff-1283-4714-a4c5-75eae08a3d8a">

**Comment by facelessuser** [comment]
Closed by: https://github.com/facelessuser/MarkdownPreview/commit/d5376c64b2065c95f90e8f614527f5ae4c23e39c
